test/ngAnimate/animationSpec.js

```diff
@@ -977,11 +1010,12 @@ describe('$$animation', function() {
       });
       inject(function($$animation, $rootScope, $animate) {
         element.addClass('four');
+        parent.append(element);
 
         var completed = false;
         $$animation(element, 'event', {
-          from: { background: 'red' },
-          to: { background: 'blue', 'font-size': '50px' }
+          from: { height: '100px' },
+          to: { height: '200px', 'font-size': '50px' }
```

Comment:
When the element is attached to a parent element then the colors are actually applied and getComputedStyle (which is what element.css in jQuery uses) actually figures out what the color is. Beforehand it was just using the inline style. Since we now deal with actually getting the RGB value for the color it was easier just to use a more stable value like a height measurement value.